source/features/conversation-activity-filter.tsx

```diff
@@ -99,8 +110,10 @@ async function handleSelection({target}: Event): Promise<void> {
 	);
 
 	// Update the state of the other dropdown
-	select(`.${dropdownClass} [aria-checked="false"][data-value="${currentSetting}"]`)!.setAttribute('aria-checked', 'true');
-	select(`.${dropdownClass} [aria-checked="true"]:not([data-value="${currentSetting}"])`)!.setAttribute('aria-checked', 'false');
+	if (pageDetect.isConversation()) {
```

Comment:
Why does this condition exist? The PR Files tab also has a sticky bar and it should also have the dropdown

![firefox](https://user-images.githubusercontent.com/1402241/124117086-e7dffe00-da99-11eb-8faf-c4ba992088dc.gif)